cloudbuild.yaml

```diff
@@ -17,23 +18,32 @@ steps:
     ln -s $$PWD $$shadow || stat $$shadow
 
     # Install ko from release.
-    curl -L -o ko.tar.gz https://github.com/google/ko/releases/download/v0.6.0/ko_0.6.0_Linux_i386.tar.gz
+    curl -L -o ko.tar.gz https://github.com/google/ko/releases/download/v0.8.2/ko_0.8.2_Linux_i386.tar.gz
     tar xvfz ko.tar.gz
     chmod +x ko
     alias ko=./ko
 
+    debug_tag=""
+    if [ "${TAG_NAME}" != "" ]; then
+      debug_tag=${TAG_NAME}-debug
+    fi
+
     # Use the ko binary to build the crane and gcrane builder images.
-    ko publish --platform=all -B github.com/google/go-containerregistry/cmd/crane -t latest -t "$COMMIT_SHA"
-    ko publish --platform=all -B github.com/google/go-containerregistry/cmd/gcrane -t latest -t "$COMMIT_SHA"
+    ko publish --platform=all -B github.com/google/go-containerregistry/cmd/crane  -t latest -t "$COMMIT_SHA" -t "$TAG_NAME"
+    ko publish --platform=all -B github.com/google/go-containerregistry/cmd/gcrane -t latest -t "$COMMIT_SHA" -t "$TAG_NAME"
 
     # Use the ko binary to build the crane and gcrane builder *debug* images.
-    export KO_CONFIG_PATH=/workspace/.ko/debug/
-    ko publish --platform=all -B github.com/google/go-containerregistry/cmd/crane -t debug
-    ko publish --platform=all -B github.com/google/go-containerregistry/cmd/gcrane -t debug
+    # These are pushed to gcr.io/go-containerregistry/{g}crane/debug:...
```

Comment:
I think we should continue to publish `gcr.io/go-containerregistry/[g]crane:debug` as well that mirrors `latest`.